recipes/fltk/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "fltk" %}
+{% set version = "1.3.4" %}
+{% set longversion = "1.3.4-2" %}
+{% set md5 = "69128be72365cbedd1d380739f0164e3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ longversion }}-source.tar.gz
+  url: http://fltk.org/pub/{{ name }}/{{ version }}/{{ name }}-{{ longversion }}-source.tar.bz2
+  md5: {{ md5 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
```

Comment:
Can you add `zlib, jpeg, png` to build and run requirements, so that it doesn't use bundled dependencies?

Reply:
You'll have to pin those libraries as well.